wiki/Ranking_criteria/en.md

```diff
@@ -129,7 +129,7 @@ Important to understand before using:
 ### Rules
 
 - **The audio file of a beatmap must...**
-  - **...use the `.mp3` or `.ogg` file format.**
+  - **...be encoded in MP3 (`.mp3`) or Ogg Vorbis (`.ogg`).**
   - **...have an average bit rate no greater than 192 kbps for `.mp3` files, or 208 kbps for `.ogg` files.**
```

Comment:
```suggestion
  - **...be encoded in MP3 (`.mp3`) or [Ogg Vorbis](https://en.wikipedia.org/wiki/Vorbis) (`.ogg`).**
  - **...have an average bit rate no greater than 192 kbps for MP3 files, or 208 kbps for Ogg Vorbis files.**
```

Reply:
Out of curiosity, where does the `208 kbit` come from? Some ogg VBR preset?

Reply:
Briefly explained when it was PR'd https://github.com/ppy/osu-wiki/pull/9266 . Basically, was meant to simplify encoding guidelines. It was difficult to just say "In X quality setting" as bitrate would be within a range.

Reply:
mmk  🤷